tailwind.config.js

```diff
@@ -20,11 +20,11 @@ module.exports = {
       typography: theme => ({
         DEFAULT: {
           css: {
+            fontSize: 'var(--jp-content-font-size1)',
```

Comment:
Does this need to be protected in a `.myst`? Or this is in the typography anyways (I think that is the case..!).

Reply:
Typography has
```
.myst {
  @apply prose prose-stone dark:prose-invert break-words max-w-none w-full overflow-auto;
}
```

Which, as I understand it, is where the MyST stylings are actually applied